tools/pip/setup.py

```diff
@@ -164,7 +159,7 @@ def skip_markdown_comments(md):
     package_data['mxnet'].append('mxnet/libquadmath.so.0')
     if os.path.exists(os.path.join(libdir, 'libopenblas.so.0')):
         shutil.copy(os.path.join(libdir, 'libopenblas.so.0'), mxdir)
-        package_data['mxnet'].append('mxnet/libquadmath.so.0')
+        package_data['mxnet'].append('mxnet/libopenblas.so.0')
```

Comment:
?

Reply:
this part looked wrong as it adds openblas in but records quadmath instead.

Reply:
Makes sense, thanks